code/datums/datum.dm

```diff
@@ -4,6 +4,8 @@
     var/list/datum_components //for /datum/components
     var/ui_screen = "home"  //for tgui
     var/use_tag = FALSE
+	var/datum/weakref/weakref

```

Comment:
Apparently /datum as defined in this code uses four spaces instead of a single tab, causing an "inconsistent indentation" compiler error with this line.

Reply:
Spaces as indentation? Oh god, it just keeps getting worse and worse.

Reply:
the silly thing that they were there before me even in original TG version of datum.

Reply:
@kevinz000

Reply:
additionally @ACCount12 fuck off